pkg/providers/vsphere/vsphere.go

```diff
@@ -57,7 +57,7 @@ var defaultClusterConfigMD string
 var defaultSecretObject string
 
 //go:embed config/defaultStorageClass.yaml
-var defaultStorageClass []byte
+var DefaultStorageClass []byte
```

Comment:
I would advice against making a package variable like this exported, it could be accidentally overwritten 
Can we just make a exported method that returns a copy of it?